src/navigation/actions.js

```diff
@@ -172,3 +181,20 @@ export const createCustomerInvoice = (otherParty, currentUser) => dispatch => {
 
   dispatch(gotoCustomerInvoice(newCustomerInvoice));
 };
+
+/**
+ * Updates a Stocktake with the passed array of itemIDs and navigates to a
+ * StocktakeEditPage.
+ *
+ * @param {Object} stocktake realm Stocktake object
+ * @param {Array}  itemIds   Array of item id strings that should be the new
+ *                           Items in the stocktake.
```

Comment:
Maybe make it more clear here what `setItemsByID` is doing

Reply:
renaming the method might be enough